app/controllers/ores_plot_controller.rb

```diff
@@ -10,6 +10,16 @@ def course_plot
     render json: json_data
   end
 
+  def delete_ores_data
```

Comment:
This should be a private method. Other than that, I think this is ready to ship.